src/Services/Forms/Fields/Wysiwyg.php

```diff
@@ -26,6 +26,8 @@ class Wysiwyg extends BaseFormField
         'bold',
         'italic',
         'underline',
+        'subscript',
+        'superscript',
```

Comment:
Same here, if the developer does not specify their options in a field, they will end up with new buttons after updating Twill, which I consider as a "breaking" change. We should remove it from this default configuration for now. We might decide to add them back for the next major release.